backend/models.py

```diff
@@ -4,15 +4,21 @@
 from pydantic import BaseModel, validator
 from enum import Enum
 from db import db
-from typing import List
+from typing import List, TypedDict
+
+
+class TranslationExample(TypedDict):
```

Comment:
Lets use this class as a temporary data structure that is created from the dictionary.